<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpisakPasaPoRasi</class>
 <widget class="QDialog" name="SpisakPasaPoRasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spisak pasa po rasi</string>
  </property>
  <widget class="QLabel" name="labelIzlozba">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Izlozba</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRasa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Rasa</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTekstUcestvovali">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ukupan broj pasa koji su ucestvovali -</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrojUcesnika">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTekstRasaUcestvovali">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ukupan broj pasa odredjene rase koji su ucestvovali -</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrojRasaUcestvovali">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrikazi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Prikazi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzadji">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Izadji</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>20</y>
     <width>151</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxIzlozba">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRasa">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
